Update configuration.html

Grubuntu opened the pull request:
Ajout des descriptions succintes des paramètres de configuration sur la page de Paramètrage avancé.

Grubuntu committed:
Update configuration.html

Ajout des descriptions succintes des paramètres de configuration

Grubuntu commented:
![Capture d’écran (150)](https://user-images.githubusercontent.com/56045316/200409911-15203424-9666-4aba-80cc-d2a5f0238105.png)
Afin de clarifier la dénomination des paramètres avancés, j'ai ajouté les courtes descriptions des paramètres que l'on retrouve sur la notice du Github de l'EcoPV.

Jetblack31 commented:
Rhaaahhh ! super, merci, ca fait un moment que je me disais qu'il fallait améliorer les choses !
Possible de n'écrire que le sigle du paramètre en gras et le reste de la description normal ?
A+

Grubuntu commented:
Oui tout est possible en html et CSS, il faut juste trouver comment hahaha. 
Je regarde ça dès que j'ai le temps pour améliorer ça, il est probable que je doive ajouter un style dans le maxpv.css

Grubuntu committed:
Update configuration.html

Modification des entêtes de cellules pour afficher seulement les paramètres en gras :

Modification des <th> (style par défaut gras) en <td> (style par défaut normal) et ajout de balise bold <b> sur le paramètre

Grubuntu commented:
![Capture](https://user-images.githubusercontent.com/56045316/200550448-363cee3b-5e19-48d1-acbb-295cb1a72de4.JPG)

Modifications effectuées Je testerai en vrai sur le routeur ce soir, mais normalement ça devrait être bon.

Jetblack31 commented:
C'est beau, merci ! 
Je fusionnerai le PR et sortirai la prochaine release le WE prochain.
A+

Jetblack31 commented:
Ajout de documentation. Merci à Grubuntu !
